pytest_unit.ini

```diff
@@ -0,0 +1,8 @@
+[pytest]
+addopts =
+    -s
+    -v
+    -rs
+    --ignore-glob='*_nose.py'
+    --ignore='yt/frontends/owls_subfind/tests/test_outputs.py'
+    --ignore='yt/frontends/ramses/tests/test_outputs.py'
```

Comment:
Why are these  specific tests skipped?

Reply:
I was told a while ago that there was an issue with ramses. As for owls, it just took absolutely forever to run, so I skipped it in the interest of time. I actually totally forgot they were skipped until seeing this comment, though.

Reply:
I don't think there is an issue with ramses (as the maintainer of the frontend). Maybe there's an issue with ramses and pytest though. In any case these tests should be revived before this goes in.

Reply:
Don't worry about ignored test at the moment. Those ones are answer tests and this PR doesn't enable any answer tests yet.

Reply:
I would still suggest we replace those ignore flags with `@pytest.mark.skip` decorators. In particular because it forces us to write down a `reason` argument, and it's harder to miss when looking a the test file itself.

Reply:
What was I was trying to say there is that those tests are not ignored, they *don't exist* yet. We're gonna follow this PR with other adding answer test on frontend by frontend basis. When times come for Ramses or owls_subfind, those entries will be removed.